### ISSUE TITLE
Running without jquery

### ISSUE THREAD

**Issue by kellyselden** [thefrontside/emberx-select#199]
Are you interested in tracking this effort? I tried to go without jQuery in our app (we aren't using it directly), but the first console error I noticed was from here. It is fair if you don't want to support this use case, in which case you can close this issue.

**Comment by cowboyd** [comment]
Totally cool supporting this use-case, is there a best practice for how do do this? 

"Just use DOM API?"

Use some other shim?

**Comment by kellyselden** [comment]
This is a pretty good guide, although slightly outdated by now http://miguelcamba.com/blog/2017/04/11/the-future-of-embers-testing-and-the-beheading-of-jquery/

There is a section at the bottom on how to update an addon. It involves using the new jquery-free test helpers (codemod available I believe), and some ember-try fixing for older ember versions. This also helps a lot https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-jquery.md.

**Comment by NullVoxPopuli** [comment]
any progress on this? just ran in to: 
![image](https://user-images.githubusercontent.com/199018/49051930-d9315c00-f1b7-11e8-8b7b-e62932cd8504.png)
So, I can't use atm.

**Comment by Robdel12** [comment]
Have you tried using the new beta? `v4.0.0-beta.2`?

**Comment by NullVoxPopuli** [comment]
yeah, pointing at master removed the error. forgot to reply back :)

**Comment by Robdel12** [comment]
Woop! Awesome. I'll close this issue when 4.x lands officially 🎉 

(also, lmk if you come across any bugs!!)

**Comment by Turbo87** [comment]
@Robdel12 any idea when v4 will be released?
